qt/python/mantidqt/widgets/plotconfigdialog/axestabwidget/presenter.py

```diff
@@ -97,6 +98,39 @@ def apply_properties(self):
 
         self.update_view()
 
+    def apply_all_properties(self):
+        """Update the axes with the user inputted properties"""
+        # Make sure current_view_props is up to date if values have been changed
+        self.axis_changed()
+        ax_names = self.axes_names_dict.keys()
+        view_props = self.current_view_props
+        for key in ax_names:
+            ax = self.axes_names_dict[key]
```

Comment:
Could this be simplified to be `for ax in self.axes_names_dict.values()` rather than accessing the dictionary twice?

Reply:
Yes the grid lines should be affected by this, I'll fix that.
I'll update the figure options documents
You're right I can change the iterator.